ovirt_imageio/client/_ui.py

```diff
@@ -50,11 +52,16 @@ def draw(self, elapsed, size, value, transferred, phase=None, last=False):
         self._write_output(line, end)
 
 
+FORMATTER = {
```

Comment:
OUTPUT_FORMAT?

We always use the term "format" and never use "formatter". Lets be consistent.